### ISSUE TITLE
Defocus every first typing

### ISSUE THREAD

**Issue by Planxnx** [FormidableLabs/use-editable#30]
When you try to type the text for the first time, it will re-render and defocus from the input box.
So you need to select the input box and type again (it's annoying)

My workaround is force re-rendering for the first load. Is there another solution to fix this problem ??

Here is an example: [code sandbox](https://codesandbox.io/p/devbox/astro-code-editor-867yh6?file=%2Fsrc%2Fcomponents%2FEditor.tsx%3A14%2C20).

**Environments**
```
- astro: 4.0.3
- @astrojs/react: 3.0.7
- react: 18.2.0
- use-editable: 2.3.3
- prism-react-renderer: 2.3.0
```

**Comment by keenbug** [comment]
TLDR: Try adding `tabIndex={-1}` to the container.

I also ran into this problem. It seems to be related to Chrome/Chromium, neither Firefox nor Safari seem to have this problem. use-editable already seems to have a workaround for this, [see the code](https://github.com/FormidableLabs/use-editable/blob/69db4747eb92f5886db24dd4aa06fbe30b73eb9c/src/useEditable.ts#L456C32-L456C32).

It seems that adding `tabIndex={-1}` is needed for the workaround to function, see BugEditor.tsx:21 in the [updated code sandbox](https://codesandbox.io/p/devbox/astro-code-editor-forked-gqvvzw?file=%2Fsrc%2Fcomponents%2FBugEditor.tsx%3A6%2C59).

**Comment by MetaMmodern** [comment]
@keenbug as a quick fix that works fine, however it completely breaks accessibility and Tab navigation.
Also, in my case I noticed this happening even with Safari browser. But in my case I wrapped the component with editable field into `React.memo()` (as an attempt to fix rerenders from parent component). The issue is not happening without `memo`

**Comment by thisispiers** [comment]
`tabIndex={0}` shouldn't break tab navigation and works fine for us in Chrome
